subsys/bluetooth/common/Kconfig

```diff
@@ -94,20 +94,37 @@ config BT_BUF_ACL_RX_COUNT
 	  When Controller to Host flow control is not enabled the Controller
 	  can assume that the Host has infinite amount of buffers.
 
+config BT_BUF_CMD_COMPLETE_EVT_SIZE
+	  int "Maximum supported HCI Command Complete Event buffer length"
+	  # LE Read Supported Commands command complete event.
+	  default 68
+	  range 68 255
+	  help
+	    Maximum supported HCI Command Complete event buffer size.
+	    This value does not include the HCI Event header. This value is
+	    used by both the Host and the Controller for buffer sizes that
+	    include HCI Command Complete events. It should be set according
+	    to the expected HCI Command Complete events that can be generated
+	    from the configuration.
+
+	    If the subset of possible HCI Command Complete events is unknown,
```

Comment:
To me this reads as "if you use a split build (ie hci on another chip), then you better use 255".
So I propose you also add `default 255 if !BT_CTLR`. Same remark for the other kconfig.

Reply:
Thanks.